zgw_consumers/contrib/setup_configuration/steps.py

```diff
@@ -0,0 +1,34 @@
+from django_setup_configuration.configuration import BaseConfigurationStep
+from django_setup_configuration.exceptions import SelfTestFailed
+
+from zgw_consumers.models import Service
+
+from .models import ServicesConfigurationModel
+
+
+class ServiceConfigurationStep(BaseConfigurationStep[ServicesConfigurationModel]):
+    """
+    Configure Services to connect with external APIs
+    """
+
+    verbose_name = "Configuration to connect with external services"
+    config_model = ServicesConfigurationModel
+    namespace = "zgw_consumers"
+    enable_setting = "zgw_consumers_config_enable"
+
+    def is_configured(self, model: ServicesConfigurationModel) -> bool:
+        return False
+
+    def execute(self, model: ServicesConfigurationModel):
+        ignore_fields = ["identifier", "connection_check_expected_status"]
+        for config in model.services:
+            Service.objects.update_or_create(
+                slug=config.identifier,
+                defaults={
```

Comment:
I don't like this because it makes future breaking changes less visible and throws away all the benefits of static type checking.

Can you please just spell out all the mapped fields/config model properties? This is the only place where we make this translation from semantic yaml to implied DB state, let's make this easy to debug.

Reply:
changed it